### ISSUE TITLE
Installing WSL Breaks Installation of Windows Feature Updates (1803, and 1903)

### ISSUE THREAD

**Issue by Bluesman74** [microsoft/WSL#4153]
For the last two feature updates I have had terrible trouble getting them installed.  Not sure what the error number was last time but this time Windows 1903 Update failed with error code 0x80070005.

This would happen whether I was requesting it through Windows Updates, or through the upgrade assistent or the Media Creation Tool.

For the 1809 Update, I had to remove Developer mode, but for this 1903 Update it was only after I removed all WSL Distributions, that the installation finally got past 94%

**Comment by benhillis** [comment]
Closing out old issues, if you're still having issues on newer versions of Windows please open a new issue.
